lib/constants.js

```diff
@@ -49,6 +49,25 @@ export const THEMES = [
       meta: '#555'
     }
   },
+  {
+    id: 'a11y-dark',
+    name: 'a11y-dark',
```

Comment:
Can we name it `A11y Dark`? `id` can stay as is.

Reply:
Happily!